course/models.py

```diff
@@ -192,6 +192,14 @@ class Course(models.Model):
             "notifications about the course."),
             verbose_name=_('Notify email'))
 
+    force_lang = models.CharField(max_length=200, blank=True, null=True,
+            default=None,
+            help_text=_(
+                "Which language is forced to be used for this course. "
+                "If not set, displayed language will be determined by "
+                "user browser preference"),
```

Comment:
* The help text should specify how that language is encoded. (likely by referencing Django's language codes)
* Whole sentences get periods at the end. :)

Reply:
> The help text should specify how that language is encoded. (likely by referencing Django's language codes)

I'm wondering is that helpful? Because the user (instructor) just see the (translated) description, there's no access to the language code in the form.

Reply:
If someone is looking at this in the admin, possibly.

Reply:
Ok. However, there's another case (for me). I used `zh-cn` instead of `zh-hans`  as `Simplified Chinese` through customized settings.  If an instructor refer to the Django document, he will be misled to a wrong languague code. So a better solution might be concatenate the language_code to the language description?

Reply:
I just realize that help_text is assuming the settings has `USE_I18n` enabled. I.e., that description can not cover when it is not enabled.

Reply:
Yeah, that sounds reasonable.